src/test/emulators/storage/files.spec.ts

```diff
@@ -157,6 +196,229 @@ describe("files", () => {
           })
         ).to.be.rejectedWith(NotFoundError);
       });
+
+      it("should throw an error if TTL in Seconds aren't passed", () => {
+        const storageLayer = getStorageLayer(ALWAYS_TRUE_RULES_VALIDATOR);
+
+        expect(
+          storageLayer.getObject({
+            bucketId: "bucket",
+            decodedObjectId: "dir%2Fobject",
+            urlSignature: "mockSignature",
+            urlUsableSeconds: getCurrentDate(),
+          })
+        ).to.be.rejectedWith(BadRequestError);
+      });
+
+      it("should throw an error if usable Seconds aren't passed", () => {
+        const storageLayer = getStorageLayer(ALWAYS_TRUE_RULES_VALIDATOR);
+
+        expect(
+          storageLayer.getObject({
+            bucketId: "bucket",
+            decodedObjectId: "dir%2Fobject",
+            urlSignature: "mockSignature",
+            urlTtlSeconds: 10,
+          })
+        ).to.be.rejectedWith(BadRequestError);
+      });
+
+      it("should throw an error if the URL has expired. TTL is 1 Second", () => {
+        const storageLayer = getStorageLayer(ALWAYS_TRUE_RULES_VALIDATOR);
+
+        expect(
+          storageLayer.getObject({
+            bucketId: "bucket",
+            decodedObjectId: "dir%2Fobject",
+            urlSignature: "mockSignature",
+            urlTtlSeconds: SIGNED_URL_MIN_TTL_SECONDS,
+            urlUsableSeconds: getAdjustedDate(-1),
+          })
+        ).to.be.rejectedWith(BadRequestError);
+      });
+
+      it("should throw an error if the URL has expired. TTL is 1 Week(Max)", () => {
```

Comment:
Isn't this testing the same thing as the previous test?

Reply:
I was testing that the max and min time both work